chains/solana/utils/common/transactions.go

```diff
@@ -175,6 +175,7 @@ func sendTransactionWithLookupTables(ctx context.Context, rpcClient *rpc.Client,
 
 	txsig, err := rpcClient.SendTransactionWithOpts(ctx, tx, rpc.TransactionOpts{SkipPreflight: skipPreflight, PreflightCommitment: rpc.CommitmentProcessed})
 	if err != nil {
+		fmt.Println(tx) // debugging if tx errors
```

Comment:
Doesn't this pollute logs too much for tests that expect errors?

Reply:
i don't think so? it would be an RPC error which is a different method - everything else would not expect errors on the initial RPC call, it would expect onchain reverts